### ISSUE TITLE
Vitastick not reconnecting to PS3 after closing

### ISSUE THREAD

**Issue by munchluxe63** [xerpi/vitastick#5]
If vitastick is closed at some point (by start+select or the PS button), input won't register while connected to a PS3 until the PSVita is rebooted.

**Comment by xerpi** [comment]
Yeah I'm aware of this problem, I wish I had some free time to look into it but unfortunately I won't until the end of June :/

**Comment by Mirlo** [comment]
Any chance of fixing this a year later?
